src/main/java/study/Number.java

```diff
@@ -0,0 +1,53 @@
+package study;
+
+import static study.StringAddCalculator.ZERO;
+
+import java.util.Objects;
+
+public class Number {
+    private final int number;
+
+    Number(int number) {
+        this.number = number;
+    }
+
+    public static Number validator(String text) {
```

Comment:
```suggestion
    public static Number create(String text) {
```
우선 static factory method 만들어 주신 것 좋네요 ㅎㅎ 👍 
validator의 역할은 값 검증 시 실패할 경우 exception 등을 해줄 것 같은데, 결과적으로 Number를 return해주어서 조금 어색하게 느껴집니다.
값은 이렇게 만들어주고 validate를 내부 private 메서드로 만들어보면 어떨까요..?